docs/_data/mesheryctlcommands/pattern/delete/flags/delete-file.yaml

```diff
@@ -0,0 +1,4 @@
+name: --file, -f
+description: delete pattern file will remove deployment of the pattern file
```

Comment:
Can we provide more information to the user? This description isn't helpful beyond what the user already assumes that "delete" does. Can you delete multiple files at-a-time? Does it happen immediately? Can you restore a file? Can you provide a pattern ID or a pattern name? Do you have to provide a file when you're trying to delete a pattern? Why can't you just provide a pattern ID/name instead of having to provide a file, too?

Reply:
![Screenshot from 2021-06-01 14-16-13](https://user-images.githubusercontent.com/75872316/120294641-05c41280-c2e4-11eb-8c98-56e0ff7eb7dc.png)

Thanks @utkarsh-pro sir, for helping me!
This takes a lengthy format, so I will make it like:
![Screenshot from 2021-06-01 14-33-17](https://user-images.githubusercontent.com/75872316/120297125-6b190300-c2e6-11eb-9825-899b71a662ce.png)